src/renderer.ts

```diff
@@ -628,6 +628,17 @@ const render = (format: Format, excludeStyles = false) =>
                 });
                 return styledH('figure', figureAttributes, [ styledH('div', attributes), figcaption ]);
             }
+
+            case ElementKind.AudioAtom: {
+                const { theme } = format;
+                const pillar = themeFromString('pillar/' + themeToPillar(theme));
```

Comment:
I'm a little puzzled by this line... there seem to be a couple of conversions back and forth?